### PR TITLE
Fix javascript error for cross domain iframe that contains iframe without src="javascript: void(0);" (close #114)

### LINKED ISSUE #114: javascript error for cross domain iframe that contains iframe without src="javascript: void(0);"
Files to reproduce:
### index.js
```javascript
var http = require('http');
var fs = require('fs');

http.createServer(function (req, res) {
    var content = fs.readFileSync('index.html');
    res.end(content);
}).listen(3000);

http.createServer(function(req, res){
    var content = fs.readFileSync('frame.html');
    res.end(content);
}).listen(3001);

```
### index.html
```javascript
<html>
	<head></head>
	<body>
		<iframe src="http://localhost:3001/iframe.html"></iframe>
	</body>
</html>
```

### iframe.html
```javascript
<html>
	<body>
	    <h1>IFrame content</h1>
		<iframe src="javascript: void(0);"></iframe>
	</body>
</html>
```

It is browser specific problem:
1)In Chrome 45.0.2454.93:
Uncaught SecurityError: Failed to read the 'frame' property from 'Window': Blocked a frame with origin "http://localhost:1402" from accessing a frame with origin "http://localhost:1401". Protocols, domains, and ports must match.
2)In Firefox 40.0.3:
TypeError: 'getAttribute' called on an object that does not implement interface Element.
3)In IE 11 the sample works without errors

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @inikulin @churkin

### CODE REVIEW

File: test/client/fixtures/utils/dom-test.js [hunk 1]
```diff
@@ -399,3 +399,19 @@ test('IsDomElement for plain object (T198784)', function () {
     strictEqual(eval(processScript('obj.target')), 'ok');
     /* eslint-enable no-unused-vars */
 });
+
+asyncTest('getParentWindowWithSrc (GH-114)', function () {
```
Comment (inikulin) [inline test/client/fixtures/utils/dom-test.js]: Test name should describe the test case (good example: https://github.com/LavrovArtem/testcafe-hammerhead/blob/master/test/client/fixtures/sandbox/iframe-test.js#L175)

**Comment by inikulin** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**